### ISSUE TITLE
Make Widevine per-site

### ISSUE THREAD

**Issue by tomlowenthal** [brave/brave-browser#3179]
Right now, Widevine is either off or on. But it would be worth letting a person decide which sites they want to let run Widevine. 
Related: #1940

**Comment by Brave-Matt** [comment]
+1 from Community -- some sites seem to require Widevine even if it's not used typically. Aliexpress for example (see user thread) prompts the user to install WV when navigating to an individual product page on the site:
https://community.brave.com/t/how-do-i-disable-widevine-prompt-aliexpress/62889/12

**Comment by spoolr** [comment]
Yes, please fix this.  Tired of per page unending notifications.

**Comment by Swazimodo** [comment]
widevine is from Google and I don't want it active everywhere. I would want to be able to enable it only for things like netflix and prime video. Because it doesn't show up in the regular extensions list I can't configure it to work only on some websites.
